.travis.yml

```diff
@@ -1,32 +1,31 @@
 env:
   global:
+    - coverage=false
     - CC_TEST_REPORTER_ID=806ffaf1bcd1080b8b6ba4839e7acefdb6508332b18162816e4683307e008fec
 
 language: php
 
 matrix:
   include:
   - php: 7.0
+    env: coverage=true
```

Comment:
Only the `PHP-7.0` version sets `coverage` is true. And code climate executes code coverage uploading.